models/user.go

```diff
@@ -131,14 +131,21 @@ type User struct {
 	// Maximum repository creation limit, -1 means use global default
 	MaxRepoCreation int `xorm:"NOT NULL DEFAULT -1"`
 
-	// Permissions
-	IsActive                bool `xorm:"INDEX"` // Activate primary email
-	IsAdmin                 bool
-	IsRestricted            bool `xorm:"NOT NULL DEFAULT false"`
+	// IsActive true: primary email is activated, user can access Web UI and Git SSH.
+	// false: an inactive user can only log in Web UI for account operations (ex: activate the account by email), no other access.
```

Comment:
```suggestion
	// false: an non-verified user, who can only log in Web UI for account operations
	// (ex: activate the account by email), and don't have any other access.
```

Reply:
As above. We do not need to introduce new concepts. Just follow the old ones.